spec/acceptance/openldap__server_spec.rb

```diff
@@ -25,8 +25,8 @@ class { 'openldap::server': }
     it 'with cert, key and ca' do
       pp = <<-EOS
         class { 'openldap::server':
-          ssl_key   => "/etc/ldap/ssl/${::fqdn}.key",
-          ssl_cert  => "/etc/ldap/ssl/${::fqdn}.crt",
+          ssl_key   => "/etc/ldap/ssl/${networking['fqdn']}.key",
+          ssl_cert  => "/etc/ldap/ssl/${networking['fqdn']}.crt",
```

Comment:
please use $facts and not topscope facts (they are deprecated):
```suggestion
          ssl_key   => "/etc/ldap/ssl/${facts['networking']['fqdn']}.key",
          ssl_cert  => "/etc/ldap/ssl/${facts['networking']['fqdn']}.crt",
```

Reply:
What?! How does this pass CI :scream:

Reply:
Oh, `${networking['fqdn']}` is just `${::networking['fqdn']}`.  I though this got removed from Puppet but it looks like it is still just deprecated :grimacing: